source/includes/_assets.md

```diff
@@ -39,59 +39,60 @@ a batch.
 ## GET /entities/{entity_id}/assets
 
 ```shell
-curl https://api.gresb.com/api/v1/entities/5028/assets \
+curl https://api.gresb.com/api/v1/entities/20028/assets \
   -H "Authorization: Bearer $ACCESS_TOKEN"
 ```
 
 > Response
 
 ```json
-[
-    {
-        "gresb_asset_id": 442,
-        "country": "NL",
-        "state_province": "Noord-Holland",
-        "city": "Amsterdam",
-        "address": null,
-        "lat": null,
-        "lng": null,
-        "construction_year": 2000,
-        "asset_ownership": 100,
-        "partners_id": "GRB_OFF_357891Z",
-        "certifications": [
-          {
-            "id": 5913,
-            "certification_id": 901,
-            "name": "Sunshine Energy A",
-            "level": "Premium",
-            "size": "415.7"          
-          }
-        ],
-        "annual_data": [
-          {
-              "year": 2020,
-              "asset_name": "GRESB HQ",
-              "asset_size": 500,
-              "property_type_code": "OCHI"    
-          },
-          {
-              "year": 2019,
-              "asset_name": "GRESB HQ",
-              "asset_size": 450,
-              "property_type_code": "OCHI"
-          },
-          {
-              "year": 2018,
-              "asset_name": "GRESB Headquarter",
-              "asset_size": 425,
-              "property_type_code": "OCHI"    
-          }
-        ],
-        "_outliers": [],
-        "created_at": "2019-01-15T11:07:13.436Z",
-        "updated_at": "2020-01-24T12:05:11.456Z"
-    }
-]
+{
+    "gresb_asset_id": 833335,
+    "country": "CD",
```

Comment:
please change it to `NL` to be consistent with a "real"(ish) location

Using the GRESB Office as location and name could be nice.